scan_matcher:
  ros__parameters:
    global_frame_id: "map"
    robot_frame_id: "base_link"
    registration_method: "NDT"
    ndt_resolution: 2.0
    ndt_num_threads: 2
    gicp_corr_dist_threshold: 5.0
    trans_for_mapupdate: 1.5
    vg_size_for_input: 0.35
    vg_size_for_map: 0.1
    use_min_max_filter: true
    scan_min_range: 1.0
    scan_max_range: 20.0
    scan_period: 0.1
    map_publish_period: 15.0
    num_targeted_cloud: 20
    set_initital_pose: false
    initital_pose_x: 0.0
    initital_pose_y: 0.0
    initital_pose_z: 0.0
    initital_pose_qx: 0.0
    initital_pose_qy: 0.0
    initital_pose_qz: 0.0
    initital_pose_qw: 0.0
    use_imu: false
    use_odom: true
    debug_flag: false
    map_path: "/home/misys/forza_ws/race_stack/state_estimation/src/maps/teras/map.pcd"

